build/lin.sh

```diff
@@ -294,16 +332,54 @@ cd ${DEPS}/vips
   --disable-debug --disable-introspection --without-analyze --without-cfitsio --without-fftw --without-heif \
   --without-imagequant --without-magick --without-matio --without-nifti --without-OpenEXR --without-openslide \
   --without-pdfium --without-poppler --without-ppm --without-radiance \
-  --with-zip-includes=${TARGET}/include --with-zip-libraries=${TARGET}/lib \
-  --with-jpeg-includes=${TARGET}/include --with-jpeg-libraries=${TARGET}/lib
+  --with-zip-includes=${TARGET}/include --with-zip-libraries=${TARGET}/lib
+# https://docs.fedoraproject.org/en-US/packaging-guidelines/#_removing_rpath
+sed -i'.bak' 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
 make install-strip
 
 # Cleanup
 rm -rf ${TARGET}/lib/{pkgconfig,.libs,*.la,cmake}
-cd ${TARGET}/include
 
-# Set RPATH to $ORIGIN
-find ${TARGET}/lib -type f -name "*.so*" -exec sh -c "patchelf --set-rpath '\$ORIGIN' --force-rpath {}" \;
```

Comment:
Note that the `patchelf` trick isn't necessary anymore. It's now handled in the `LDFLAGS` above.